web/core/components/cycles/list/cycle-list-item-action.tsx

```diff
@@ -81,7 +81,11 @@ export const CycleListItemAction: FC<Props> = observer((props) => {
   // derived values
   const cycleStatus = cycleDetails.status ? (cycleDetails.status.toLocaleLowerCase() as TCycleGroups) : "draft";
   const showIssueCount = useMemo(() => cycleStatus === "draft" || cycleStatus === "upcoming", [cycleStatus]);
-  const transferableIssuesCount = cycleDetails ? cycleDetails.total_issues - cycleDetails.completed_issues : 0;
+  const transferableIssuesCount = cycleDetails
+    ? cycleDetails.total_issues -
```

Comment:
@mathalav55 could you please move the logic to a separate method and reuse it rather than repeating it?